### PR TITLE
Tb 3059 nav bar documentation

### PR DESCRIPTION
**Pull request by pro100svitlo**
### What 🕵️ 🔍

- This PR contains some minor changes in the `nav_bar` related code comments
- Added documentation to the `NavBar` related classes

----------

### How to test 🥼 🔬

- check the docs :smile:

----------

### References 📝 🔗

- [jira task](https://xainag.atlassian.net/browse/TB-3059)

----------

### CODE REVIEW

File: docs/NAV_BAR.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      like
+    ]);
+  }
+```
+
+In this way you can describe different content of the `NavBar` for different screens. And then
+[NavBarObserver] will take care of changing those configs when `navigation` happens between screens.
+
+If you need to notify `NavBar` that config was changed for the current screen 
+(f.e. `isLiked` become `true`), provide those changes in `NavBarConfig get navBarConfig`
```
Comment (peterstojanowski) [inline docs/NAV_BAR.md:86]: `f.e.` -> `i.e.`

File: docs/TOOLTIP.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
-`Tooltip` allows you to show a tooltip messages, something like Android toasts, but in a dart way :
-smile:
+`Tooltip` allows you to show a tooltip messages, something like Android toasts, but in a dart way :smile:
```
Comment (peterstojanowski) [inline docs/TOOLTIP.md:1]: `show a tooltip messages` -> `show tooltip messages`

**Review by peterstojanowski — COMMENTED** [review]
Looks really good 👍. Just two tiny comments.

File: docs/TOOLTIP.md (hunk 2 above)
Comment (pro100svitlo) [reply docs/TOOLTIP.md:1]: fixed 😉

File: docs/NAV_BAR.md (hunk 1 above)
Comment (pro100svitlo) [reply docs/NAV_BAR.md]: fixed 😉

**Review by peterstojanowski — APPROVED** [review]

**Pull request merged**